common/state-machine-lib/src/main/java/org/eclipse/dataspaceconnector/common/statemachine/StateMachine.java

```diff
@@ -50,7 +50,7 @@ private StateMachine(String name, Monitor monitor, ExecutorInstrumentation instr
         this.name = name;
```

Comment:
Since it's not scope of this PR, I would exclude these modification on this `StateMachine.java` file

Reply:
it's just one line... do you really want to open a separate PR for that? and it annoyed me very much :)